src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -202,4 +204,25 @@ export class ServicesComponent extends BaseComponent<IServicesComponentPropertie
 
         return ListComponent.renderColumn(textToRender || "", ListComponent.defaultColumnRenderer, colDataClassName);
     }
+
+    private static _getServiceStatus(service: IServiceItem, cssClassName: string): React.ReactNode {
+        let statusProps: IStatusProps = Statuses.Success;
+        let tooltipText: string | undefined;

```

Comment:
not sure undefined is really needed

let tooltipText: string = "";

above line may be better

Reply:
Done. Modified.